[wam][chromium68] Choose external and upstream backend with PCKGCONF

msisov opened the pull request:
By default, the recipe uses the external wayland backend (aka intel
based) when chromium is built, but it's possible to choose
the backend now.

Add the following lines to local.conf (or local.dev.inc) to use
upstream wayland backend:

PACKAGECONFIG_append_pn-chromium68 = " use-upstream-wayland"

Signed-off-by: Maksim Sisov <msisov@igalia.com>

jdapena commented on recipes-wam/chromium/chromium68.bb:
```diff
@@ -71,6 +71,16 @@ PACKAGECONFIG[jumbo] = "use_jumbo_build=true jumbo_file_merge_limit=${JUMBO_FILE
 PACKAGECONFIG[lttng] = "use_lttng=true,use_lttng=false,lttng-ust,lttng-tools lttng-modules babeltrace"
 
+
+PACKAGECONFIG ?= "use-upstream-wayland"
```
By default, we are still not shipping this. So let's remove this line. When added, it should be altogether with the line declaring jumbo by default.

msisov replied:
By default, we use ozone_platfrom_wayland_external. And when one uses “use-upstream-wayland”, the ozone_platfrom_wayland_external must be set to false. Thus, this packageconfig is needed

jdapena replied:
No, it does not work like this. PACKAGECONFIG[use-upstream-wayland] uses left side if PACKAGECONFIG includes the string use-upstream-wayland, and the right side if it is not present.
This line does nothing right now because there is another PACKAGECONFIG ?= before for soft-setting jumbo. So, when this is evaluated, PACKAGECONFIG already has a value, and will do nothing.
We just need to remove this line for now. When we set the use-upstream-wayland option as default, we will need to modify the line 56 (the one that soft-sets jumbo) to also set use-upstream-wayland.

msisov replied:
sorry. I confused the things :)) now it's fixed.

jdapena reviewed (changes requested):
Do not acronymize in commit message. Better to have PACKAGECONFIG than PCKGsomething.

msisov committed:
[wam][chromium68] Choose external and upstream backend with PCKGCONF

By default, the recipe uses the external wayland backend (aka intel
based) when chromium is built, but it's possible to choose
the backend now.

Add the following lines to local.conf (or local.dev.inc) to use
upstream wayland backend:

PACKAGECONFIG_append_pn-chromium68 = " use-upstream-wayland"

Signed-off-by: Maksim Sisov <msisov@igalia.com>
